templates/react/field-types/Vec/detail/render.hbs

```diff
@@ -0,0 +1,6 @@
+<div>
+  <h4>{{title_case field_name}}</h4>
+  {#each {{variable_to_read}} as el}
+    <span>{{> (concat field_type.type "/" widget "/detail/render") variable_to_read="el"}}</span>
```

Comment:
```suggestion
    <span key={ i }>{{> (concat field_type.type "/" widget "/detail/render") variable_to_read="el"}}</span>
```

This resolves a [React warning](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key) saying that every JSX item in a list should have a `key` property. (The linked page says that JSX items rendered using a `list.map()` loop _must_ have a key. It also says that they way I did it, just autogenerating a key using something as impermanent as an array index, is a crappy way of doing it and defeats the purpose. I'm not clear on what the purpose actually is 🤷🏼‍♂️ )